src/content/docs/plugin/deep-linking.mdx

```diff
@@ -128,27 +130,63 @@ For [universal links](https://developer.apple.com/documentation/xcode/allowing-a
 }
 ```
 
-Where `$DEVELOPMENT_TEAM_ID` is the value defined on `tauri.conf.json > tauri > bundle > iOS > developmentTeam` or the `TAURI_APPLE_DEVELOPMENT_TEAM` environment variable and `$APP_BUNDLE_ID` is the value defined on `tauri.conf.json > identifier`. See [applinks.details](https://developer.apple.com/documentation/bundleresources/applinks/details) for more information.
+:::note
+The response `Content-Type` header must be `application/json`.
+
+The `.well-known/apple-app-site-association` endpoint must be served over HTTPS.
+To test localhost you can either use a self-signed TLS certificate and install it on the iOS simulator or use services like [ngrok].
+:::
+
+Where `$DEVELOPMENT_TEAM_ID` is the value defined on `tauri.conf.json > tauri > bundle > iOS > developmentTeam` or the
+`TAURI_APPLE_DEVELOPMENT_TEAM` environment variable and `$APP_BUNDLE_ID` is the value defined on [`tauri.conf.json > identifier`].
+
+To verify if your domain has been properly configured to expose the app associations, you can run the following command,
+replacing `<host>` with your actual host:
+
+```sh
+curl -v https://app-site-association.cdn-apple.com/a/v1/<host>
+```
+
+See [applinks.details](https://developer.apple.com/documentation/bundleresources/applinks/details) for more information.
 
-#### App
+### Desktop
 
-You also need to add the associated domains to your app's `entitlements` file:
+On Linux and Windows deep links are delivered as a command line argument to a new app process.
+The deep link plugin has integration with the [single instance] plugin if you prefer having a unique app instance receiving the events.
 
-```xml title="src-tauri/gen/apple/[App Name]_iOS/[App Name]_iOS.entitlements" ins={5-9}
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-	<key>com.apple.developer.associated-domains</key>
-	<array>
-		<string>applinks:your.website.com</string>
-		<string>applinks:nother.site.br</string>
-	</array>
-</dict>
-</plist>
+- First you must add the `deep-link` feature to the single instance plugin:
+
+```toml title="src-tauri/Cargo.toml"
+[target."cfg(any(target_os = \"macos\", windows, target_os = \"linux\"))".dependencies]
+tauri-plugin-single-instance = { version = "2.0.0-rc", features = ["deep-link"] }
+```
+
+- Then configure the single instance plugin **after** the deep link plugin:
```

Comment:
```suggestion
- Then configure the single instance plugin **after** the deep link plugin but **before** _all_ other plugins:
```
maybe something like this too just to be sure?

Reply:
According to Tony it doesn't as this is only relevant for the primary app which has enough time to set up both plugins.

So i think it makes sense to keep single-instance as the very first plugin to keep it inline with its own docs (for the sake of the argument, imagine that this is actually documented there) and to make sure it closes the instances as soon as possible.